cmd/fetchd/cmd/capricornmigrate.go

```diff
@@ -0,0 +1,390 @@
+package cmd
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	"github.com/fetchai/fetchd/app"
+	"github.com/spf13/cobra"
+	tmjson "github.com/tendermint/tendermint/libs/json"
+	tmtypes "github.com/tendermint/tendermint/types"
+
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/errors"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/cosmos-sdk/x/genutil/types"
+	ibctransfer "github.com/cosmos/cosmos-sdk/x/ibc/applications/transfer/types"
+	ibchost "github.com/cosmos/cosmos-sdk/x/ibc/core/24-host"
+	ibccoretypes "github.com/cosmos/cosmos-sdk/x/ibc/core/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+const (
+	flagGenesisTime                    = "genesis-time"
+	flagInitialHeight                  = "initial-height"
+	flagConsensusBlockMaxBytes         = "consensus-block-max-bytes"
+	flagConsensusBlockMaxGas           = "consensus-block-max-gas"
+	flagFoundationAddress              = "foundation-address"
+	flagFoundationTokensToBurn         = "foundation-tokens-to-burn"
+	flagStakingParamsHistoricalEntries = "staking-historical-entries"
+	flagBridgeNewContractPath          = "bridge-new-contract-path"
+	flagMobixNewContractPath           = "mobix-new-contract-path"
+)
+
+const (
+	bridgeContractCodeID = 2
+	mobixContractCodeID  = 3
+)
+
+var (
+	// contractCodeIDsToRemove list the codeIDs of the contracts to be removed by the migration
+	contractCodeIDsToRemove = []string{
+		"1", // Unused bridge contract (effective one use codeID 2)
+		"4", // Reconciliation contract (not needed anymore)
+	}
+)
+
+// AddCapricornMigrateCmd returns a command to migrate genesis to stargate version.
+func AddCapricornMigrateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "capricorn-migrate [genesis-file]",
```

Comment:
On line 68 it looks like this argument is required but the usage here gives me the impression that it's options, should this be `<genesis-file>` instead of `[genesis-file]`?

Reply:
Yep good point thanks! I'll also need to add the --bridge-new-contract-path and --mobix-new-contract-path flags that are also required.